src/xrpl-account.js

```diff
@@ -61,6 +61,11 @@ class XrplAccount {
         return (await this.getInfo()).MessageKey;
     }
 
+    async getDomain() {
+        const domain = (await this.getInfo()).Domain;
```

Comment:
`const domain = (await this.getInfo())?.Domain;`

Since `this.getInfo()` can return undefined or null. This will suppress the error when accessing the object.